plugins/removeHiddenElems.js

```diff
@@ -123,6 +135,32 @@ exports.fn = (root, params) => {
   return {
     element: {
       enter: (node, parentNode) => {
+        if (
+          (node.name === 'style' && node.children.length !== 0) ||
+          hasScripts(node)
+        ) {
+          deoptimized = true;
+          return;
+        }
+
+        // Store uses so that they can be removed if broken
```

Comment:
Mind dropping the new comments?

Comments that just explain what the code is doing are redundant, this also applies to most of the other comments in the project.

The code is clear enough as it is. If it's really so complicated that it needs an explanation, it should be in its own function and get a JSDoc, or alternatively may need a refactor.